cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -378,33 +381,65 @@ teeTraceChainTipElide
   -> Tracer IO (WithSeverity (ChainDB.TraceEvent blk))
 teeTraceChainTipElide = elideToLogObject
 
-traceChainInformation :: Trace IO Text -> ChainInformation -> IO ()
-traceChainInformation tr ChainInformation { slots, blocks, density, epoch, slotInEpoch } = do
-  -- TODO this is executed each time the chain changes. How cheap is it?
-  meta <- mkLOMeta Critical Confidential
-  let tr' = appendName "metrics" tr
-      traceD :: Text -> Double -> IO ()
-      traceD msg d = traceNamedObject tr' (meta, LogValue msg (PureD d))
-      traceI :: Integral a => Text -> a -> IO ()
-      traceI msg i = traceNamedObject tr' (meta, LogValue msg (PureI (fromIntegral i)))
-
-  traceD "density"     (fromRational density)
-  traceI "slotNum"     slots
-  traceI "blockNum"    blocks
-  traceI "slotInEpoch" slotInEpoch
-  traceI "epoch"       (unEpochNo epoch)
+traceChainInformation
+  :: forall blk. ConvertRawHash blk
+  => Trace IO Text
+  -> ChainInformation blk
+  -> IO ()
+traceChainInformation tr chainInfo = do
+    -- TODO this is executed each time the chain changes. How cheap is it?
```

Comment:
This is probably more correct:

```suggestion
    -- TODO this is executed each time the tip changes. How cheap is it?
```

Reply:
Or "chain tip" for extra redundancy :trollface: .